docker/backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.16.0-buster-slim AS builder
+FROM node:17.9.1-buster-slim AS builder
```

Comment:
Is the change to the Dockerfile intended?

I don't see the relation to the PR so should be reverted.